.changeset/little-pets-perform.md

```diff
@@ -0,0 +1,6 @@
+---
+"openapi-typescript-docs": minor
```

Comment:
Just a note: We don’t need to version the docs; they always get released on merge to `main`. Should make sure these are ignored from changesets.